brigade/core/configuration.py

```diff
@@ -96,6 +102,14 @@ def __init__(self, config_file=None, **kwargs):
         for k, v in kwargs.items():
             setattr(self, k, v)
 
+        inventory = (self._resolve_import_from_string(kwargs.get("inventory")) or
```

Comment:
this can be rewritten as:

```
inventory = self._resolve_import_from_string(kwargs.get("inventory", self.inventory))
```

which is a bit cleaner